pandas/core/common.py

```diff
@@ -2072,7 +2072,7 @@ def _random_state(state=None):
     elif isinstance(state, np.random.RandomState):
         return state
     elif state is None:
-        return np.random.RandomState()
+        return np.random.mtrand._rand
```

Comment:
Can we use something like this instead?
```python
seed = np.random.randint(2 * 32)
return np.random.RandomState(seed)
```

That way we don't need to use NumPy's private API.